newIDE/app/src/GameDashboard/GameDashboardCard.js

```diff
@@ -0,0 +1,657 @@
+              disabled={disabled}
+              style={styles.projectFilesButton}
+            />
+          </Tooltip>
+        </>
+      )}
+    </Line>
+  );
+
+  const renderLastModification = (i18n: I18nType) =>
+    projectFileMetadataAndStorageProviderName ? (
+      <LastModificationInfo
+        file={projectFileMetadataAndStorageProviderName}
+        lastModifiedInfo={lastModifiedInfo}
+        storageProvider={itemStorageProvider}
+        authenticatedUser={authenticatedUser}
+        currentFileMetadata={currentFileMetadata}
+        textColor="secondary"
+        textPrefix={isCurrentProjectOpened ? null : <Trans>Last edited:</Trans>}
+      />
+    ) : game ? (
+      <LineStackLayout noMargin expand>
+        <Text color="secondary" noMargin size="body-small">
+          <Trans>Last edited:</Trans>
+        </Text>
+        <Text color="secondary" noMargin size="body-small">
+          {i18n.date(game.updatedAt * 1000)}
+        </Text>
+      </LineStackLayout>
+    ) : null;
+
+  const renderStorageProvider = (i18n: I18nType) => {
+    const icon = itemStorageProvider ? (
+      itemStorageProvider.renderIcon ? (
+        itemStorageProvider.renderIcon({
+          size: 'small',
+        })
+      ) : null
+    ) : (
+      <WarningRound />
+    );
+    const name = itemStorageProvider ? (
+      i18n._(itemStorageProvider.name)
+    ) : isCurrentProjectOpened ? (
+      <Trans>Project not saved</Trans>
+    ) : (
+      <Trans>Project not found</Trans>
+    );
+
+    return (
+      <Line noMargin alignItems="center">
+        {icon && (
+          <>
+            {icon}
+            <Spacer />
+          </>
+        )}
+        <Text noMargin color="secondary">
+          {name}
+        </Text>
+      </Line>
+    );
+  };
+
+  const renderThumbnail = () => (
+    <GameThumbnail
+      gameName={gameName}
+      gameId={game ? game.id : undefined}
+      thumbnailUrl={gameThumbnailUrl}
+      background="light"
+      width={
+        isMobile
+          ? undefined
+          : // On medium/large screens, adapt the size to the width of the window.
+            Math.min(245, Math.max(130, window.innerWidth / 4))
+      }
+    />
+  );
+
+  const buildOpenContextMenu = (
+    i18n: I18nType,
+    projectsList: FileMetadataAndStorageProviderName[]
+  ): Array<MenuItemTemplate> => {
+    const actions = [];
+    if (projectsList.length > 1) {
+      actions.push(
+        ...[
+          ...projectsList.map(fileMetadataAndStorageProviderName => {
+            return {
+              label: getProjectItemLabel(
+                fileMetadataAndStorageProviderName,
+                storageProviders,
+                i18n
+              ),
+              click: () => onOpenProject(fileMetadataAndStorageProviderName),
+            };
+          }),
+          { type: 'separator' },
+          {
+            label: i18n._(t`See all in the game dashboard`),
+            click: game ? () => onOpenGameManager({ game }) : undefined,
```

Comment:
To disable if no game ?

Reply:
yeah this case shouldn't happen in theory, cause a project can't be linked to another one without a game. But I'll disable it just in case